docs/formation/bootstrap.md

```diff
@@ -0,0 +1,32 @@
+# Governance Bootstrap
+
+In March 2023, the CCC community identified a need for improved organizational structure to better enable contributions to the standard.
+
+This improvement work is being done in two phases, detailed below.
+
+Because this work includes the creation of strict charters and limitations on future change, it will only happen this way once. All subsequent changes must follow steps outlined in the respective group charters.
+
+# Phase 1 — Steering Committee Bootstrapping
+
+The first, and most important step, is to create a charter and assign members to the top level organizational unit: the Steering Committee. This must be done through a consensus of the most active CCC participants, with an opportunity for input from the entire community.
```

Comment:
@eddie-knight - I have a concern with: "...a consensus of the most active CCC participants". The steerco will become the standard's oversight body and to have credibility with regulators it needs to be Financial Services. Broader community input is critical (e.g. from vendors, CSPs, etc), but we need Banks/FinServ overseeing it to ensure regulators will take it seriously.

Reply:
I agree with this. 💯

Reply:
Agree. We need to ensure this is open to input from the broader community. But from the standards and regulatory perspective it would need to be focussed on financial institutions to gain that credibility from regulators. The FI's also have that deep understanding of the regulatory scope to help guide the project.

Reply:
changing to _most active CCC participants **from financial service institutions**_

Reply:
changing to "_most active CCC participants **from financial service institutions**_"